src/pages/hotel.js

```diff
@@ -3,155 +3,138 @@ import Layout from '@components/layout'
-    height: '0',
-    '& iframe': {
-      left: '0',
-      top: '0',
-      height: '100%',
-      width: '100%',
-      position: 'absolute'
-    }
+  overflow: 'hidden',
+  paddingBottom: '56.25%',
+  position: 'relative',
+  height: '0',
+  '& iframe': {
+    left: '0',
+    top: '0',
+    height: '100%',
+    width: '100%',
+    position: 'absolute'
+  }
 })
 
 const HotelPage = () => {
-    const { site } = useStaticQuery(
-        graphql`
-            query {
-                site {
-                    siteMetadata {
-                        longDates
-                    }
-                }            
-            }`
+  const { site } = useStaticQuery(graphql`
+    query {
+      site {
+        siteMetadata {
+          longDates
+          venue
+        }
+      }            
+    }
+  `)
+
+  const metadata = site.siteMetadata
+  const dates = metadata.longDates
+  const venue = metadata.venue
+  const isOpen = bookingStatus === "open"
+  const isClosed = bookingStatus === "closed"
+
+  if (!isOpen) {
+    return (
+      <Layout>
+        <Seo title="Hotel" />
+        <PageHeader title="Hotel" />
+        <PageContent>
+          {isClosed ? (
+            <p>
+              <br />
+              Online hotel booking is now closed. Please <Link to="/contact">contact us</Link> if you would like assistance booking a room at our discounted rates.
+              <br />
+            </p>
+          ) : (
+            <p>
+              <br />
+              {/* This message also gets displayed if the booking status is invalid */}
+              Online hotel booking is not yet available. Stay tuned, we'll announce hotels soon!
+              <br />
+            </p>
+          )}
+        </PageContent>
+      </Layout>
     )
+  }
 
-    const dates = site.siteMetadata.longDates
-    
   return (
-  <Layout>
-    <Seo title="Hotel" />
-     
-    <PageHeader 
-      title="Hotel" 
-    />
+    <Layout>
+      <Seo title="Hotels" />
+      <PageHeader title="Hotel" />
+      <PageContent>
+
+        <img
+          src={hotelPicture}
+          alt='' // This is a presentational image, so the alt should be an empty string
+        />
 
-    <PageContent>
-      <StaticImage 
-        src="../images/Hotel.jpeg"
-        // This is a presentational image, so the alt should be an empty string
-        alt=''
-      />
-      <p>
-        U.S. IdolFes will be held on {dates}, at the Sonesta Redondo Beach &amp; Marina. Details on how to access the venue are available below.
-          {/* Use when hotel booking is closed  */}
-          {/*
-           <br /><br />
-          
-          Online hotel booking is now closed. Please <Link to="/contact">contact us</Link> if you would like assistance booking a room at our discounted rates. 
-           <br /><br />*/}
+        <p>U.S. IdolFes will be held on {dates}, at the {venue}. {hotelNote || ''}</p>
+        <p>Details on how to access the venue are available below.</p>
```

Comment:
This information should not be displayed when hotel booking is closed. The dates won't be decided yet, and it's possible (albeit unlikely) the venue could change. The hotel note also means our staff need to keep track of what information needs to be on this page in their heads, instead of it being in the CMS. 

We really need to control this with the 3 main states the page may be in.